third_party/boringssl/src/crypto/bio/file.c

```diff
@@ -597,16 +615,16 @@ static int MS_CALLBACK file_gets(BIO *bp, char *buf, int size) {
   buf[0] = '\0';
 #if defined(OPENSSL_SYS_STARBOARD)
   ret = -1;
-  SbFileInfo info;
-  SbFileGetInfo((SbFile)bp->ptr, &info);
-  int64_t current = SbFileSeek((SbFile)bp->ptr, kSbFileFromCurrent, 0);
-  int64_t remaining = info.size - current;
+  struct stat info;
+  fstat(((FilePtr)bp->ptr)->fd, &info);
+  int64_t current = lseek(((FilePtr)bp->ptr)->fd, SEEK_CUR, 0);
```

Comment:
This should be `lseek(((FilePtr)bp->ptr)->fd, 0, SEEK_CUR);`?

Reply:
Yep good catch